src/bindings/jna/libelektra/src/main/java/org/libelektra/plugin/NativePlugin.java

```diff
@@ -25,6 +25,7 @@
 	 * Constructor for loading an Elektra plugin
 	 * @param pluginName The plugin name
 	 * @param errorKey The errorKey
+	 * @param modules #3713 add parameter description and update other
```

Comment:
Description seems wrong.

Reply:
Java doc is WIP... - I'm currently reworking the java doc

Reply:
Will be addressed via #3754